### ISSUE TITLE
IdsForm: Accessibility WCAG Guideline: 4.1.2 Name, Role, Value Defects

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1337]
Describe the bug: Error message is not automatically announced by screen reader.

To Reproduce:
"1. Launch the URL: https://main.wc.design.infor.com/ids-form/example.html
2. Hit enter on submit button, without filling or by filling the edit fields with wrong details and verify using screen reader"
3. When we hit enter on submit button without filling or by filling the edit fields with wrong details then the error message which appears is not automatically announced by screen reader. 
  
Expected behavior:
When we hit enter on submit button without filling or by filling the edit fields with wrong details then the error message which appears should be automatically announced by screen reader.

User Impact:
If error message will not be announced automatically then screen reader user will not be able to identify whether an error has occurred or not.

Screenshots: https://www.screencast.com/t/PxNJmM8aKF

Dev Recommendation:
"Provide aria-live=""""assertive"""" and role=""""alert"""" to the error message to make an announcement for screen reader users.
 Refer to:
 [https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/ARIA](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/ARIA19)19"

Platform
    Windows 10/Chrome/NVDA

Additional context
NA
